tests/manage/pv_services/test_pvc_resize.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.framework.testlib import skipif_ocs_version
+
+log = logging.getLogger(__name__)
+
+
+@skipif_ocs_version(['>4.1', '<4.5'])
+@pytest.mark.parametrize(
+    argnames=["interface"],
+    argvalues=[
+        pytest.param(
+            constants.CEPHBLOCKPOOL, marks=pytest.mark.polarion_id("OCS-00000")
```

Comment:
@ebenahar could you please assign someone to create the Polarion test cases?

Reply:
Looks like we have already OCS-325:

https://polarion.engineering.redhat.com/polarion/#/project/OpenShiftContainerStorage/workitem?id=OCS-325

Reply:
@humblec could you please change accordingly?

Reply:
@ebenahar missed it, sorry. I have updated the patchset. PTAL.